tests/fixtures/tests.py

```diff
@@ -620,6 +620,40 @@ def test_output_formats(self):
             format='xml', natural_foreign_keys=True
         )
 
+    def test_loading_with_excludes(self):
+        Site.objects.all().delete()
+
+        management.call_command('loaddata', 'fixture1', exclude=['fixtures'], verbosity=0)
+        self.assertFalse(Article.objects.exists())
+        self.assertFalse(Category.objects.exists())
+        self.assertQuerysetEqual(Site.objects.all(), [
+            '<Site: example.com>',
+        ])
+
+        Site.objects.all().delete()
+        Article.objects.all().delete()
+        Category.objects.all().delete()
+
+        management.call_command('loaddata', 'fixture1', exclude=['fixtures.Article'], verbosity=0)
+        self.assertFalse(Article.objects.exists())
+        self.assertQuerysetEqual(Category.objects.all(), [
+            '<Category: News Stories>',
+        ])
+        self.assertQuerysetEqual(Site.objects.all(), [
+            '<Site: example.com>',
+        ])
+
+    def test_exclude_option_errors(self):
+        # Excluding a bogus app should throw an error.
+        with self.assertRaisesMessage(management.CommandError,
+                                      "No installed app with label 'foo_app'."):
```

Comment:
We try to avoid non-4 space indent like this.  You could move this to `msg = "..."` instead.